### ISSUE TITLE
About the official website

### ISSUE THREAD

**Issue by gigarlew** [StripesFramework/stripes#80]
Hi guys, just found the official website http://www.stripesframework.org/ is redirected to a suspicious website.

**Comment by Uzume** [comment]
- https://web.archive.org/web/20090302191844/http://stripes.mc4j.org/confluence/display/stripes/Home
- https://web.archive.org/web/20140916061357/http://stripesframework.org/display/stripes/Home
- https://stripesframework.atlassian.net/wiki/spaces/STRIPES/overview

**Comment by ojaha065** [comment]
Seems that their domain registration expired and some bot just grabbed it.

**Comment by vdeconinck** [comment]
Yeah, plus the atlassian Wiki seems to be abandoned too because I get hammered every week with the same newsletter saying the most popular article is [Read JSON Requests with Stripes 1.5](https://stripesframework.atlassian.net/wiki/spaces/STRIPES/pages/53379776/Read+JSON+Requests+with+Stripes+1.5), which states since 2017 that "In Stripes 1.7 there is a much better solution", Except 1.7 is not "released"... So I guess the only reason that article is popular is because it's in the newsletter...

Is anybody still using Stripes ? If not, what did you migrate to ?
